samples/bluetooth/mesh/dfu/distributor/README.rst

```diff
@@ -221,12 +221,12 @@ SMP over Bluetooth authentication
 =================================
 
 By default, the SMP characteristics don't require authentication when using SMP over Bluetooth to access the :ref:`management subsystem <zephyr:mcu_mgr>`.
```

Comment:
```suggestion
By default, the SMP characteristics do not require authentication when using SMP over Bluetooth to access the :ref:`management subsystem <zephyr:mcu_mgr>`.
```

Reply:
This is unrelated to my change. @peknis, could you raise a PR aligning this part?

It takes a lot of effort to make CI green for such complex upgrades and there is a lot of stuff in the pipeline for the next release. I would like to avoid unnecessary CI runs as part of this PR.

Reply:
We can take this later as this is not a crucial thing.